### DISCUSSION TITLE
model.yml file structure

### DISCUSSION THREAD

**Discussion by Daannoa** [EdgeTX/edgetx#1583] [Q&A] [answered]
I'm currently having fun writing a little python program which aims to copy some data  (screandata, topnardata, logicalSw,…) from one model.yml file to another model.yml.
To achieve this I use the "ruamel" module.
My first attempts seem promising but I still notice that the order of the configuration options is not always respected in the files that I generate.
The options that I add to a file are found at the end of the tree.
Below an example with the options "curves" and "points".

Is it annoying?
Does the order of configuration options matter?

Thanks for your help.
![YMLstructure](https://user-images.githubusercontent.com/93767482/153492461-02343106-16c6-49af-95e0-8296f46beda8.JPG)

**Comment by pfeerick** [comment, marked as answer]
I do not believe so. I believe the whole file is read in and then parsed,
and the same in reverse when writing (whole file is written out when saving
any changes). So radio firmware should be fine with any order changes, and
will re-order the file how it likes it on the next change anyway.
